db/migrate/20211209022316_create_tracks.rb

```diff
@@ -1,20 +0,0 @@
-class CreateTracks < ActiveRecord::Migration[6.1]
-  def change
-    create_table :tracks do |t|
-      t.string :name
-      t.text :description
-      t.string :source_link
-      t.date :release_date
-      t.time :duration
-      t.string :countries
```

Comment:
don't remove migration, thatis already in common for other developer branch
You should write new migration, to delete this table